media-libs/portsmf/files/portsmf-239-revert-extern-to-static-change.patch

```diff
@@ -0,0 +1,48 @@
+From 149f4687db15d1759723d5d3e8334980239f9d56 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Arsen=20Arsenovi=C4=87?= <arsen@aarsen.me>
+Date: Sun, 18 Dec 2022 16:09:27 +0100
+Subject: [PATCH 3/3] Revert "fix linking Alg_atoms symbol_table with dynamic
+ linking with MSVC"
+
+This reverts commit 951b636f7d0cba370d483a91f1897c71f3d98530.
+
+This change separated the symbol_table that portSMF provides from the one that
+the consumers use.  I suspect this was wrong, and the problem it fixes in
+actuality was somewhere else.
+---
+This effectively makes the packaged version v238.
+
+Forwarded: https://codeberg.org/tenacityteam/portsmf/pulls/11
```

Comment:
I usually prefer this up the top, but it's no big deal.

Reply:
I like it here, usually, since it doesn't change the rest of the mboxy structure, but nothing inherently changes indeed